src/api.ts

```diff
@@ -16,3 +18,18 @@ export function init(options: InitOptions) {
 }
 
 export { client };
+
+export async function getApiVersion(): Promise<string> {
+  const { data } = await client.get('/status');
+  return data.version;
+}
+
+export async function quote(chainId: number, marketId: string, operation: string, data: any) {
+  const resp = await client.post(`/v1/markets/${chainId}/${marketId}/${operation}`, data);
+  return common.classifying(resp.data);
```

Comment:
[Oral discussion] Would like to understand how `common.classifying` works.

Reply:
The main purpose of `common.classifying` is to convert `TokenAmountsObject`, `TokenAmountObject`, and `TokenObject` into Class instances for convenient use in the subsequent operations.

You can refer to: https://github.com/dinngo/protocolink-js-sdk/blob/master/packages/common/src/tokens/transform.ts#L44

Reply:
Understood.

Reply:
A follow-up question: Are there any parameters in `<QuoteAPIResponseBody<LeverageQuotation>>` that need to be classified? Seems that `quotation`, `logics`, `approvals` and `permitData` don't have those objects.

Reply:
Indeed, currently, the ones that have the opportunity to be "classifying" are logics.
However, it is not expected that users will use them directly, so there is no need for data to be "classifying."
I will remove it.